app/services/place_hold_form/builder.rb

```diff
@@ -35,7 +35,7 @@ def holdables_present?(bib_info)
       return false if bib_info.call_list.blank?
 
       @call_list = bib_info.call_list.map { |call| Call.new record: call }
-      filter_holdables if @call_list.count > 1
+      filter_holdables if @call_list.count.positive?
```

Comment:
This would _not_ solve the problem you encountered yesterday though, right? Because the call list was empty on that made-up catkey execution would never get past the guard statement at the top. 

This change is for a different scenario. It is challenging the assumption that when there is a call list there is at least 1 holdable call. Is that right?

Reply:
It solves the problem I had yesterday, eg. catkey: 473854, this record is withdrawn and call list is not empty, has 1 holding.

Reply:
ah I thought it had an empty call list

Reply:
yeah, it has one call with library key `ZREMOVED`

Reply:
ok fair enough. it does seem like a slim chance of happening in-real-life, but guess why not

Reply:
honestly very unlikely, like maybe if the url is saved and used again later and then in the meantime the record would changed to withdrawn - sounds almost no way but still I thought it would not hurt to check that one holding :shrug: